pkg/swagger/generator/template_repo.go

```diff
@@ -487,6 +489,29 @@ func padSurround(entry, padWith string, i, ln int) string {
 	return strings.Join(res, ",")
 }
 
+// padDocument indent multi line document with given pad
+func padDocument(str string, pad string) string {
+	// get the OS name
+	// set the appropriate line separator
+	var lineSeparator string
+	if runtime.GOOS == "windows" {
```

Comment:
Ref: https://github.com/open-policy-agent/conftest/blob/40d4f223b2052cc32766b46c0e6cbc8f3c54ac63/parser/yaml/yaml.go#L36

Reply:
fixed.

That will suit the cases when `\r\n`  are explicitly declared in the openapi spec file.